coops-ui/src/App.vue

```diff
@@ -226,6 +228,9 @@ export default {
     storeEntityInfo (response) {
       if (response && response.data && response.data.business) {
         this.setEntityName(response.data.business.legalName)
+        // TODO: Replace placeholder with `response.data.business.legalType` when legalTypes are seeded.
```

Comment:
@severinbeauvais Definitely agree but without the data being seeded it will cause errors. Are legalTypes values existing in Production? We currently don't have control over any of the real data, so should we wait until entities have a type assigned? A lot of our future work depends on entities having an assigned type.

Reply:
Definitely agree. A lot of work going forwards depends on legalTypes having values. Until that value is seeded in production i'm not sure how else to move forward, when the only repo we have is to PR into bcgov/master.  Should we wait until that is done before moving forward?